resources/views/admin/layout.blade.php

```diff
@@ -66,6 +64,42 @@
                     <a href="index3.html" class="nav-link">{{ trans('contents.admin.home') }}</a>
                 </li>
             </ul>
+            <ul class="navbar-nav ml-auto">
+                <li class="nav-item dropdown">
+                    @switch(Lang::locale())
+                        @case(config('constance.lang.en'))
```

Comment:
```suggestion
                        @case (config('constance.lang.en'))
```
bên duới tương tự nhá